roles/micado_master/start/files/egi/configure

```diff
@@ -1,15 +1,7 @@
-# OpenStack settings:
-export OS_PROTOCOL="openid"
-export OS_PROJECT_ID="{{ project_id }}"
-export OS_AUTH_URL="{{ auth_url }}"
-export OS_IDENTITY_API_VERSION=3
-export OS_IDENTITY_PROVIDER="{{ identity_provider }}"
-export OS_AUTH_TYPE="v3oidcaccesstoken"
-
-# EGI AAI Check-In settings:
-export CHECKIN_CLIENT_ID="{{ client_id }}"
-export CHECKIN_CLIENT_SECRET="{{ client_secret }}"
-export CHECKIN_REFRESH_TOKEN="{{ refresh_token }}"
-export CHECKIN_AUTH_URL="{{ oidc_url }}"
-
-export OS_TOKEN=$(python3 /var/lib/micado/terraform/submitter/pyGetScopedToken.py)
\ No newline at end of file
+OS_PROJECT_ID="{{ project_id }}"
```

Comment:
Is this a python file now? If so it is missing the .py extension

Reply:
After templating, I am writing it to a file with .py currently. I will change this also if that is better.